.grype.yml

```diff
@@ -0,0 +1,26 @@
+# List of vulnerabilities to ignore for the anchore scan
```

Comment:
this file looks to be the same as the one on template-infra so i think we can remove it, since it'll be copied over from template-infra when the project installs template-infra

Reply:
I actually need to tweak the infra version since it has a python specific recast in it, the `certifi` line. Should I remove this file and leave it in infra, or keep it and tweak the infra version?

Reply:
oh i see. so you're suggesting that we move the python specific thing from the infra repo to this repo? that makes sense.

Reply:
Yeah, as we continue to build in this repo, there will probably be other things we need to recast or safelist that are just for this repo, so I think we should keep the config files here since they'd fix those issues